File: src/main/java/de/rubixdev/enchantedshulkers/asm/AugmentableContainerTarget.java
package de.rubixdev.enchantedshulkers.asm;

import de.rubixdev.enchantedshulkers.Utils;
import net.minecraft.item.Item;

@SuppressWarnings("unused") // used in de.rubixdev.enchantedshulkers.asm.EnumInjector
public class AugmentableContainerTarget extends EnchantmentTargetMixin {
    @Override
    public boolean isAcceptableItem(Item item) {
        return Utils.canAugment(item);
    }
}

File: src/main/kotlin/de/rubixdev/enchantedshulkers/asm/AugmentableContainerTarget.kt
package de.rubixdev.enchantedshulkers.asm

import de.rubixdev.enchantedshulkers.Utils
import net.minecraft.item.Item

class AugmentableContainerTarget : EnchantmentTargetMixin() {
    override fun isAcceptableItem(item: Item?) = Utils.canAugment(item)
}
